src/contrib/cluster/Akka.DistributedData/Serialization/ReplicatedDataSerializer.cs

```diff
@@ -89,7 +89,7 @@ public override byte[] ToBinary(object obj)
                 case IPNCounterDictionary pn: return SerializationSupport.Compress(ToProto(pn));
                 case IPNCounterDictionaryDeltaOperation pnd: return ToProto(pnd.Underlying).ToByteArray();
                 case IORMultiValueDictionary m: return SerializationSupport.Compress(ToProto(m));
-                case IORMultiValueDictionaryDeltaOperation md: return ToProto(md.Underlying).ToByteArray();
+                case IORMultiValueDictionaryDeltaOperation md: return ToProto(md).ToByteArray();
```

Comment:
Fixed the serialization of `IORMultiValueDictionaryDeltaOperation`s by having them explicitly include the `WithValueDelta` setting that will be passed into the underlying collection.